cookbook-api/database/migrations/RunMigrations.ts

```diff
@@ -1,27 +1,29 @@
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
-const  { QueryTypes } = require('sequelize');
-const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.json')[env];
+import {QueryTypes} from 'sequelize';
+import {getSequelizeInstance} from '../SequelizeFactory';
+import * as path from 'path';
+import * as fs from 'fs';
 
-let sequelize;
-if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
-} else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
+interface Exists{
+    exists: boolean;
+}
+interface ScriptName {
+    script_name: string;
 }
 
+const basename = path.basename(__filename);
+
+var sequelize = getSequelizeInstance();
 const runDbUp = async () => {
-    const hasScriptsHistoryTable = await sequelize.query(
+
+    const hasScriptsHistoryTable = await sequelize.query<Exists>(
```

Comment:
Kind of annoying you have to have a specific `Exists` object just for one boolean field, but I get it.

Reply:
agreed- but eh- I still think it's an upside.